### PR TITLE
Add lansitec devices

### PR DESCRIPTION
**Pull request by zhangquancom**
#### Summary

Add lansitec devices:
  - badge-tracker
  - indoor-bluetooth-gateway
  - macro-bluetooth-gateway
  - container-tracker
  - helmet-sensor
  - temperature-humidity-sensor
  - precision-platinum-sensor

#### Changes

- ...
- ...

#### Checklist for Reviewers

- [ ] Title and description should be descriptive (Not just a serial number for example).
- [ ] `vendorProfileID` should not be `vendorID`.
- [ ] All devices should be listed in the vendor's `index.yaml` file.
- [ ] At least 1 image per device and should be transparent.

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/lansitec/profile-as923.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+vendorProfileID: 0
```
Comment (LDannijs) [inline vendor/lansitec/profile-as923.yaml:1]: Could you make sure all the profiles have unique `vendorProfileID`'s? I can see you have 7 so number them from 0 to 6 for example.

Comment (zhangquancom) [reply vendor/lansitec/profile-as923.yaml:1]: The vendorProfileID has been modified from 0 to 6

Comment (LDannijs) [reply vendor/lansitec/profile-as923.yaml:1]: Thank you for that 👍 By the way, you can change multiple files in 1 commit. This helps to compact the history.

Comment (zhangquancom) [reply vendor/lansitec/profile-as923.yaml:1]: Thank you for your suggestion

**Review by Jaime-Trinidad — APPROVED** [review]

**Pull request merged**